pyproject.toml

```diff
@@ -57,7 +58,17 @@ profile = "black"
 
 [tool.pytest.ini_options]
 minversion = "7.0"
-addopts = "--tb=short -s"
+addopts = '--tb=short -s --color=yes --cov-report=xml --cov=acquire --maxfail=5 --log-cli-level=0 --doctest-glob="*.pyi" --doctest-glob="*.md"'
 # log_format = "%(asctime)s %(levelname)s %(message)s"
 # log_date_format = "%Y-%m-%d %H:%M:%S"
 log_cli = true # when true, messages are printed immediately
+
+[tool.ruff]
+exclude = [
+    "__init__.py"
+]
+
+ignore = [
+    "F403",
+    "F405",
+]
```

Comment:
I'm OK to start with these excludes and ignores, though I'd prefer to have none of them in the long term.